widgets/icon/icon.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+Widget Name: Icon
+Description: An iconic icon.
+Author: SiteOrigin
+Author URI: https://siteorigin.com
+*/
+
+class SiteOrigin_Widget_Icon_Widget extends SiteOrigin_Widget {
+
+	function __construct() {
+
+		parent::__construct(
+			'sow-icon',
+			__( 'SiteOrigin Icon', 'so-widgets-bundle' ),
+			array(
+				'description' => __( 'An icon widget.', 'so-widgets-bundle' )
+			),
+			array(),
+			false,
+			plugin_dir_path( __FILE__ )
+		);
+	}
+
+	function initialize_form() {
+		return array(
+			'icon' => array(
+				'type'  => 'icon',
+				'label' => __( 'Icon', 'so-widgets-bundle' ),
+			),
+
+			'color' => array(
+				'type'  => 'color',
+				'label' => __( 'Color', 'so-widgets-bundle' ),
+			),
+
+			'size' => array(
+				'type'  => 'measurement',
+				'label' => __( 'Size', 'so-widgets-bundle' ),
+			),
+
+			'url' => array(
+				'type'  => 'link',
+				'label' => __( 'Destination URL', 'so-widgets-bundle' ),
+			),
+
+			'new_window' => array(
+				'type'    => 'checkbox',
+				'default' => false,
+				'label'   => __( 'Open in a new window', 'so-widgets-bundle' ),
+			),
+		);
+	}
+
+	function get_style_name( $instance ) {
+		return 'sow-icon';
+	}
+
+	function get_less_variables( $instance ) {
+		$les = ';asda';
```

Comment:
Let's me put breakpoints in the `get_less_variables` function. I guess I should clean up after myself. :sweat_smile: